### PR TITLE
add send-chinatown-love seller to seeds.rb

### PR DESCRIPTION
**Pull request by juliexxia**
- [ ] get square location id

### CODE REVIEW

**Comment by juliexxia** [comment]
@Jeremy-Xue have you had a chance to rustle up a square_location_id?

**Comment by Jeremy-Xue** [comment]
> @Jeremy-Xue have you had a chance to rustle up a square_location_id?

1GDT5F6VR86GP

Yup! So I generated this one we can use to test it out (I didn't know what address to put so I put mine) but I think if we want to use it in prod we should probably choose another address.

File: db/seeds.rb [hunk 1]
```diff
@@ -24,6 +24,13 @@
     website_url: nil,
     menu_url: nil,
     square_location_id: 'E4R1NCMHG7B2Y'
+  },
+  {
+      seller_id: 'send-chinatown-love',
+      accept_donations: false,
+      sell_gift_cards: false,
+      founded_year: 2020,
+      square_location_id: "" # TODO
```
Comment (jtmckibb) [inline db/seeds.rb:33]: This should probably be an ENV variable

Comment (juliexxia) [reply db/seeds.rb:33]: Should I also make the one for shunfa an ENV var?

**Pull request merged**